### PR TITLE
Publishing projects to twitter.

### PR DESCRIPTION
**Pull request by prodex**
#35 

To process previously created projects, you need to run:
```
yii project/add-share-jobs
```

### CODE REVIEW

**Comment by prodex** [comment]
moment, I delete the conflict

**Comment by prodex** [comment]
Ready!

File: migrations/m170813_061427_add_published_to_twitter.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+use yii\db\Migration;
+
+class m170813_061427_add_published_to_twitter extends Migration
+{
+    public function safeUp()
+    {
+        $this->addColumn('{{%project}}', 'published_to_twitter', $this->boolean()->defaultValue(false)->notNull());
+        $this->createIndex('idx-project-status-published_to_twitter', '{{%project}}', ['status', 'published_to_twitter']);
```
Comment (prodex) [inline migrations/m170813_061427_add_published_to_twitter.php]: Small spam in twitter) Fixed.

**Comment by samdark** [comment]
Merged. Made some minor corrections: https://github.com/samdark/yiipowered/commit/afde678ee8f6b88503b48d876ddbc3b44cf4125d

Setting it up... Thank you!

**Comment by samdark** [comment]
https://twitter.com/yiipowered

**Comment by prodex** [comment]
Cool!

**Pull request closed without merging**